lib/rules/comma-style.js

```diff
@@ -62,6 +62,49 @@ module.exports = {
         }
 
         /**
+         * Modified text based on the style
+         * @param {string} styleType Style type
+         * @param {string} text Source code text
+         * @returns {string} modified text
+         * @private
+         */
+        function getReplacedText(styleType, text) {
+            switch (styleType) {
+                case "between":
+                    return `,${text.replace("\n", "")}`;
```

Comment:
@mysticatea I din't quite get what you trying to say here.

I have a unit test covering the above use case, and the fixed code would be 

```js
let obj = {
    a, // a comment
    b
}
```